lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -2477,9 +2477,7 @@ def _resolve_marketplace_image_version(
                     node_runbook.location, node_runbook.marketplace
                 )
 
-    def _add_image_features(self, node_space: schema.NodeSpace) -> None:
-        # Load image information, and add to requirements.
-
+    def _get_marketplace_image_locations(self) -> List[str]:
```

Comment:
`_find_marketplace_image_location` is better to describe the logic.

Reply:
Done